rules/falco_rules.yaml

```diff
@@ -3398,6 +3398,21 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+- rule: Archive and Compression Activity
+  desc: Detects archive and compression activity using tar, zip, gzip, and bzip2.
+  condition: >
```

Comment:
the condition seems a bit off. It msised the usual evt.type to restrict the syscall you want to get or a macro. I guess you need spawned_process in this case. In additon I suggest to use lists for proc.name which would let the condition readable.